package/src/utils/has-integration.ts

```diff
@@ -21,10 +39,29 @@ import type { HookParameters } from "astro";
  */
 export const hasIntegration = ({
 	name,
+	position,
+	relativeTo,
 	config,
-}: {
-	name: string;
-	config: HookParameters<"astro:config:setup">["config"];
-}): boolean => {
-	return !!config.integrations.find((integration) => integration.name === name);
+}: HasIntegrationParams): boolean => {
+	const integrationPosition = config.integrations.findIndex((integration) => integration.name === name);
+
+	// Integration is not installed
+	if (integrationPosition === -1) return false;
+
+	// Not a relative check, the integration is present.
+	if (position === undefined) return true;
+
+	if (relativeTo === undefined) throw new AstroError(
```

Comment:
Not super familiar with this class. 1st argument is the error message and 2nd is a hint iirc?

Reply:
Than astro formats it nicely
![image](https://github.com/florian-lefebvre/astro-integration-kit/assets/11063910/296e6da3-f38e-4afc-bd90-b4617d21614f)
![image](https://github.com/florian-lefebvre/astro-integration-kit/assets/11063910/b2954b7b-8159-4be6-b33b-ec813ae82896)